packages/web/localizations/ko.json

```
@@ -432,14 +432,14 @@
     "signupDummyHistoryTitle2": "자산 입금 방법",
     "signupDummyHistoryMessage2": "Osmosis로 자금을 이체하는 방법 알아보기",
     "signupDummyHistoryTitle3": "거래를 시작하는 방법",
-    "signupDummyHistoryMessage3": "이 빠른 자습서를 통해 몇 분 안에 거래할 수 있습니다.",
+    "signupDummyHistoryMessage3": "이 빠른 가이드를 통해 몇 분 안에 거래할 수 있습니다.",
     "outboundTransferHistoryTitle": "Osmosis 아웃바운드 전송",
     "outboundTransferHistoryMessage": "양",
     "poolExitedHistoryTitle": "Osmosis 풀 종료됨",
     "poolExitedHistoryMessage": "풀 ID",
-    "poolJoinedHistoryTitle": "삼투 풀 합류",
+    "poolJoinedHistoryTitle": "Osmosis 풀 합류",
```

```suggestion
    "poolJoinedHistoryTitle": "오스모시스 풀 합류",
```
match expression "삼투" to "오스모시스". only this on is written in English.